app/utility/transforms/DateUtils.js

```diff
@@ -74,3 +74,12 @@ export const restInSentenceFormat = (millis) => {
         return seconds + " sec rest";
     }
 };
+
+export const checkDateWithinRange = (range, time) => {
```

Comment:
- [ ] Add comment that the range is in days, and that the time is in milliseconds